test/integration/multikueue/multikueue_test.go

```diff
@@ -1367,6 +1367,128 @@ var _ = ginkgo.Describe("Multikueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 		})
 	})
 
+	ginkgo.It("Should create a pod group on worker if admitted", func() {
+		podgroup := testingpod.MakePod("test-pod-in-group", managerNs.Name).
+			Queue(managerLq.Name).
+			ManagedByKueueLabel().
+			KueueFinalizer().
+			KueueSchedulingGate().
+			MakeGroup(3)
+
+		for _, p := range podgroup {
+			gomega.Expect(managerTestCluster.client.Create(managerTestCluster.ctx, p)).Should(gomega.Succeed())
+		}
+
+		// any pod should give the same workload Key
+		createdWorkload := &kueue.Workload{}
+		wlLookupKey := types.NamespacedName{Name: workloadpod.PodGroupName(&podgroup[0].ObjectMeta), Namespace: managerNs.Name}
```

Comment:
```suggestion
		wlLookupKey := types.NamespacedName{Name: groupName, Namespace: managerNs.Name}
```

Like we are doing on other places.